src/xo-app/backup/overview/index.js

```diff
@@ -201,6 +204,8 @@ export default class Overview extends Component {
     }
   )
 
+  _getPredicate = schedules => schedules != null
```

Comment:
This method does not return a predicate: it *is* a predicate (a function returning a boolean).